app/views/shared/_buyer_menu.html.erb

```diff
@@ -1,4 +1,4 @@
-<% unless request.host == site_account.admin_domain %>
+<% unless site_account.external_domain == request.host %>
```

Comment:
Hmm, the original one is `admin`, but the change is `domain` ? :thinking: Doesn't look right.